src/components/Announcements/AnnouncementsList.vue

```diff
@@ -1,21 +1,39 @@
 <template>
   <div class="scroll-container announcements-container" v-bind:style="{ 'maxHeight': '20rem' }">
-      <div v-for="a in allAnnouncements" v-bind:key="a.id" class="announcement-wrapper">
+      <div v-for="a in announcements" v-bind:key="a.id" class="announcement-wrapper">
         <h3>{{a.title}}</h3>
+        <p>{{ toStringDate(a.created) }}</p>
         <p>{{a.description}}</p>
       </div>
   </div>
 </template>
 
 <script>
-import { mapGetters } from 'vuex';
+import moment from 'moment';
+import api from '../../api/api';
 
 export default {
   name: 'AnnouncementsList',
-  computed: {
-    ...mapGetters('announcements', {
-      allAnnouncements: 'getAllAnnouncements',
-    }),
+  props: {
+    number: Number,
+  },
+  data() {
+    return {
+      announcements: [],
+    };
+  },
+  methods: {
+    async getAnnouncements() {
+      const res = await api.getSitewideAnnouncements({});
+      return res.announcements;
+    },
+    toStringDate(date) {
```

Comment:
Should this be a generalized util? I imagine there's many other places we'll need this function. 
It may be best to put it in a util file and unit test it.

Reply:
I don't see a generalized util file anywhere, just specific ones for different tasks. Should I make a general one?

Reply:
Yeah, maybe call it like DateUtils or StringUtils